apps/daimo-mobile/src/debugLog.ts

```diff
@@ -0,0 +1,31 @@
+const n = 400;
+const logs: string[] = [];
+
+// Hooks console.log, saves a rolling log of the last n lines.
+export function initDebugLog() {
+  const oldLog = console.log;
+  console.log = (...args) => {
```

Comment:
nice! should you be hooking into `console.error` as well here?

Reply:
good call, will do a followup for `console.warn`/`console.error`